### DISCUSSION TITLE
Some trouble with lerna-symlinked node_modules watching for v2

### DISCUSSION THREAD

**Discussion by u8sand** [parcel-bundler/parcel#5429] [Q&A]
Firstly, thanks for the awesome tool; I spent way too much time trying to figure out the best way to set up a javascript project prior to discovering parcel. Now I just use parcel; I'm really happy to see it improving. I've so-far stuck with v1 because I'm already familiar with its quirks, but i'm keen on getting v2 working for a new project I'm building. It's a monorepo project that uses lerna to bootstrap dependencies.

At the start I had an issue that the react HMR reloading wasn't working at all but it turned I had react ^15 installed and this feature only works with ^16; instead of an error message, however, I got a strange effect where only the first index.js (included from index.html) would trigger reloads, and any file imported from there would not. Just something that might be improved in the docs/code that handles HMR assertion.

My current issue at hand involves the fact that I include *local* packages that are right next to parcel and get symlinked into node_modules. I'd like that if I were to modify the contents of the dependency, parcel would detect the change and refresh; this seems to work in v1 but not in v2. I did see some issues complaining about node_modules being watched in v1 and suspect this may have been configured by default differently in v2 for performance reasons but I'm hoping there is a configuration option to re-enable it; at least for specific dependencies.

Thank you in advance, I also wondered if there was a IRC, gitter, or discord available for parcel that I could not find.

**Comment by krnlde** [comment]
I'm having the same issue with yarn workspaces in 2.0.0-nightly.551. I read something that v2 is intentionally not watching `node_modules/`. Maybe workspaces are a point against that approach. I would love to have a flag for that.

**Comment by mischnic** [comment]
Files behind symlinks that go outside the project aren't watched at the moment: https://github.com/parcel-bundler/parcel/issues/4950.

**Comment by krnlde** [comment]
Current tracking issue is #4332
